source/Playnite/HdrUtilities.cs

```diff
@@ -0,0 +1,166 @@
+﻿using Playnite.Native;
+using Playnite.SDK;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Playnite
+{
+    public class HdrUtilities
+    {
+        private static readonly ILogger logger = LogManager.GetLogger();
+
+        /// <summary>
+        /// Determines if HDR is supported on the primary display
+        /// </summary>
+        /// <returns>True if HDR is supported on the primary display, false if unknown or not supported</returns>
+        public static bool IsHdrSupported()
+        {
+            try
+            {
+                DISPLAYCONFIG_PATH_TARGET_INFO? targetInfo = GetPrimaryDisplayTargetInfo();
+                if (!targetInfo.HasValue)
+                {
+                    logger.Error("Failed to retrieve primary display target info");
+                    return false;
+                }
+
+                (bool isHdrSupported, _) = GetHdrStatus(targetInfo.Value);
+                return isHdrSupported;
+            }
+            catch (Exception e) when (!PlayniteEnvironment.ThrowAllErrors)
+            {
+                logger.Error(string.Format("Failed to check if HDR is supported: {0}", e.Message));
```

Comment:
I'm really sorry that I missed this, it will be the last thing to change, I swear.

To log an exception, do it this way instead:
```
logger.Error(e, "Failed to check if HDR is supported.");
```

It will automatically include formatted exception info together with the message.

Reply:
Done